cpp/src/reductions/scan/scan_inclusive.cu

```diff
@@ -23,6 +23,7 @@
 #include <cudf/detail/gather.hpp>
 #include <cudf/detail/iterator.cuh>
 #include <cudf/detail/null_mask.hpp>
+#include <cudf/detail/structs/utilities.hpp>
```

Comment:
Similar question here. Why do we need this?

Reply:
Same as above